src/scss/variables/_colors.scss

```
@@ -1,17 +1,20 @@
 // Color Array
 $colors: (
-    "white":white,
-    "light-shade":#e6e6e6,
+    "white":white,              // Menu button font color
+    "light-shade":#e6e6e6,      // Menu row background color
```

```suggestion
    "light-shade":#e6e6e6,
```
Removing comment, as this is not the only place this is, or can be, used.